README.md

```diff
@@ -52,13 +52,24 @@ require one for your use case.
 * [virtualenv](https://pypi.org/project/virtualenv/)
 * [venv](https://docs.python.org/3/library/venv.html)
 
+NOTE: MacOS sometimes has issues with installing since it comes with python2.7 by default.
```

Comment:
```suggestion
**NOTE**: For MacOS users, we advise installing in a virtual environment to avoid interfering with OS-level site packages.
```

It does not matter the Python version, we should tell people not to install Ape outside a virtual env unless they are using some simpler linux box